zebROS_ws/src/behaviors/src/2023_align_to_grid.cpp

```diff
@@ -0,0 +1,231 @@
+// so theoretically the best way to do this is to use the values in 2023_grid_locations.yaml to drive to the location knowing our field-relative location using an apriltag.
+// that's just a particle filter.
+// which we don't trust yet.
+// so we can do the equivalent by calculating the offset between a detected apriltag (see 2023_apriltag_locations.yaml) and the desired location
+// and averaging if there are multiple.
+// find closest apriltag to desired location, calculate offset, call path_to_apriltag
+#include "ros/ros.h"
+#include "field_obj/Detection.h"
+#include "geometry_msgs/Point.h"
+#include <ros/ros.h>
+#include <actionlib/server/simple_action_server.h>
+#include "behavior_actions/AlignToGrid2023Action.h"
+#include "behavior_actions/PathToAprilTagAction.h"
+#include <actionlib/client/simple_action_client.h>
+#include <tf2/LinearMath/Quaternion.h>
+#include <tf2_geometry_msgs/tf2_geometry_msgs.h>
+
+inline geometry_msgs::Point operator-(const geometry_msgs::Point& lhs, const geometry_msgs::Point& rhs) {
```

Comment:
inline does essentially nothing for functions in a .cpp file, so it can be removed here

Reply:
ok, thanks!